recipes/landlab/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.5.2" %}
+
+package:
+  name: landlab
+  version: {{ version }}
+
+source:
+  url: https://github.com/landlab/landlab/archive/v{{ version }}.tar.gz
+  sha256: 130dd5281254444c74ce252272db9f8cc3f9df78de42bc0538edbc899ca92c42
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+  host:
+    - python
+    - pip
+    - cython
+    - numpy 1.11.*
+  run:
+    - python
+    - cython
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - xarray
+    - matplotlib
+    - numpydoc
```

Comment:
I don't think `numpydoc` is needed, unless you are building the docs.

Reply:
Correct.